### ISSUE TITLE
object null is not iterable (cannot read property Symbol(Symbol.iterator))

### ISSUE THREAD

**Issue by zahoorahmadshah** [microsoft/vscode-docker#2572]
Does this occur consistently? 
Repro steps:

1.
2.

Action: vscode-docker.configure
Error type: TypeError
Error Message: object null is not iterable (cannot read property Symbol(Symbol.iterator))

Version: 1.9.0
OS: win32
OS Release: 10.0.18363
Product: Visual Studio Code
Product Version: 1.52.0
Language: en

<details>
<summary>Call Stack</summary>

```
NetCoreGatherInformationStep.prompt extension.bundle.js:924:29753
```

</details>

**Comment by bwateratmsft** [comment]
@zahoorahmadshah can you share information about your project? e.g. project file, dotnet version, etc.?

**Comment by bwateratmsft** [comment]
I think I know one way this could be happening; I'll put in a fix in the form of a better error message.

**Comment by bwateratmsft** [comment]
This is now available in Docker extension version 1.9.1.
